script/check-config.sh

```diff
@@ -73,10 +73,11 @@ wrap_color() {
 }
 
 wrap_good() {
-	echo "$(wrap_color "$1" white): $(wrap_color "$2" green)"
+	echo "$(wrap_color "$1" white): $(wrap_color "$2" green) ${3:-}"
```

Comment:
If we use `shift`, we can use `"$@"` to get this extra functionality somewhat more safely:

```suggestion
	local name="$1"
	shift
	local value="$1"
	shift
	echo "$(wrap_color "$name" white): $(wrap_color "$value" green)" "$@"
```